DRIVERS/VME_16Z002/vme4l-core.h

```diff
@@ -149,6 +149,7 @@ typedef void *VME4L_BRIDGE_HANDLE;
 #else
 #define DEBUG_DEFAULT 0
 #endif
+static int debug = DEBUG_DEFAULT;  /**< enable debug printouts */
```

Comment:
I'm strongly against this. What if a file including vme4l-core.h has a debug variable for something else?

Reply:
In all vme4l*c files there's debug = DEBUG_DEFAULT defined, so isn't it simpler to define it in vme4l-core.h?

Reply:
no, because vme4l-core.h may (and actually is) included by different drivers